transformers/shared/constants/signature.go

```diff
@@ -27,6 +27,7 @@ func DealSignature() string                { return getLogNoteTopicZero(dealMeth
 func DentSignature() string                { return getLogNoteTopicZero(dentMethod()) }
 func DenySignature() string                { return getLogNoteTopicZero(denyMethod()) }
 func FlapKickSignature() string            { return getEventTopicZero(flapKickMethod()) }
+func FlipFileCatSignature() string         { return getEventTopicZero(flipFileCatMethod()) }
```

Comment:
I think that we'd want to use `getLogNoteTopicZero` method here instead since the file method has the note modifier, instead of emitting a custom event.

That would make the returned signature: `0xd4e8be8300000000000000000000000000000000000000000000000000000000`